controllers/tasks.js

```diff
@@ -53,15 +53,23 @@ const fetchTasks = async (req, res) => {
  */
 const getUserTasks = async (req, res) => {
   try {
-    const { status } = req.query
+    let { status } = req.query
     const { username } = req.params
     let allTasks = []
 
-    if (status && !Object.values(TASK_STATUS).includes(status)) {
+    if (status && !Object.keys(TASK_STATUS).includes(status.toUpperCase())) {
       return res.boom.notFound('Status not found!')
     }
 
-    allTasks = await tasks.fetchUserTasks(username, status ? [status] : [])
+    if (status) {
+      if (status === 'active') {
+        status = ['active', 'blocked', 'pending', 'IN_PROGRESS', 'BLOCKED', 'SMOKE_TESTING']
+      } else {
```

Comment:
can we use constants here?

Reply:
Hi @swarajpure 
If we make them constants there is going to be collisions of the names because of this 👇 
[constants/tasks.js](https://github.com/Real-Dev-Squad/website-backend/blob/f2aa4b36123c09d465372e89f8ce8f167eed1c61/constants/tasks.js)

We can have a separate list of constants starting with something like **OLD_ACTIVE** but it might create confusions 
So what do you think we should do here ( Also one point in time old status will longer be in use and we might need to remove all of it)